acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/ValueTransferAcceptanceTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.signing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.GAS_PRICE;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.INTRINSIC_GAS;
+import static tech.pegasys.ethsigner.tests.dsl.utils.Offset.NO_OFFSET;
+
+import tech.pegasys.ethsigner.jsonrpc.response.JsonRpcError;
+import tech.pegasys.ethsigner.jsonrpc.response.JsonRpcErrorResponse;
+import tech.pegasys.ethsigner.tests.AcceptanceTestBase;
+import tech.pegasys.ethsigner.tests.dsl.Account;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+import org.junit.Test;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.utils.Convert;
+import org.web3j.utils.Convert.Unit;
+
+public class ValueTransferAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  public void valueTransfer() throws IOException {
+    final String recipient = "0x1b00ba00ca00bb00aa00bc00be00ac00ca00da00";
+    final BigInteger transferAmountWei = Convert.toWei("1.75", Unit.ETHER).toBigIntegerExact();
+    final BigInteger startBalance = ethNode().accounts().balance(recipient);
+    final Transaction transaction =
+        Transaction.createEtherTransaction(
+            richBenefactor().address(),
+            richBenefactor().getNextNonceAndIncrement(),
+            GAS_PRICE,
+            INTRINSIC_GAS,
+            recipient,
+            transferAmountWei);
+
+    final String hash = ethSigner().transactions().submit(transaction);
+    ethNode().transactions().awaitBlockContaining(hash);
+
+    final BigInteger expectedEndBalance = startBalance.add(transferAmountWei);
+    final BigInteger actualEndBalance = ethNode().accounts().balance(recipient);
+    assertThat(actualEndBalance).isCloseTo(expectedEndBalance, NO_OFFSET);
+  }
+
+  @Test
+  public void valueTransferFromAccountWithInsufficientFunds() throws IOException {
+    final String recipientAddress = "0x1b11ba11ca11bb11aa11bc11be11ac11ca11da11";
+    final BigInteger senderStartBalance = ethNode().accounts().balance(richBenefactor());
+    final BigInteger recipientStartBalance = ethNode().accounts().balance(recipientAddress);
+    final BigInteger transferAmountWei = senderStartBalance.multiply(BigInteger.TEN);
```

Comment:
can we just do a sendStartBalance.add(BigInteger.One) - multiply COULD fail if the start balance was 0 ... which it's not ... but ...

Reply:
In theory yes, in practice I had that line previously and there were intermittent failure (i.e. balance passed when it should have failed)

Reply:
Will try again, now `isEqualTo()` is being used